test/e2e/user-actions-benchmark.js

```diff
@@ -0,0 +1,142 @@
+      await driver.press('#password', driver.Key.ENTER);
+
+      await driver.clickElement('.account-menu__icon');
+      const timestampBeforeAction = new Date();
+      await driver.clickElement({ text: 'Create Account', tag: 'div' });
+      await driver.fill('.new-account-create-form input', '2nd account');
+      await driver.clickElement({ text: 'Create', tag: 'button' });
+      await driver.waitForSelector({
+        css: '.currency-display-component__text',
+        text: '0',
+      });
+      const timestampAfterAction = new Date();
+      loadingTimes = timestampAfterAction - timestampBeforeAction;
+    },
+  );
+  return loadingTimes;
+}
+
+async function confirmTx() {
+  let loadingTimes;
+  await withFixtures(
+    {
+      fixtures: 'imported-account',
+      ganacheOptions,
+    },
+    async ({ driver }) => {
+      await driver.navigate();
+      await driver.fill('#password', 'correct horse battery staple');
+      await driver.press('#password', driver.Key.ENTER);
+
+      await driver.clickElement('[data-testid="eth-overview-send"]');
+
+      await driver.fill(
+        'input[placeholder="Search, public address (0x), or ENS"]',
+        '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
+      );
+
+      const inputAmount = await driver.findElement('.unit-input__input');
+      await inputAmount.fill('1000');
+
+      await inputAmount.press(driver.Key.BACK_SPACE);
+      await inputAmount.press(driver.Key.BACK_SPACE);
+      await inputAmount.press(driver.Key.BACK_SPACE);
+      await driver.wait(async () => {
+        const sendDialogMsgs = await driver.findElements(
+          '.send-v2__form div.dialog',
+        );
+        return sendDialogMsgs.length === 1;
+      }, 10000);
+
+      await inputAmount.fill('1');
+
+      await driver.clickElement({ text: 'Next', tag: 'button' });
+      const timestampBeforeAction = new Date();
+      await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+      await driver.clickElement('[data-testid="home__activity-tab"]');
+      await driver.wait(async () => {
+        const confirmedTxes = await driver.findElements(
+          '.transaction-list__completed-transactions .transaction-list-item',
+        );
+        return confirmedTxes.length === 1;
+      }, 10000);
+
+      await driver.waitForSelector('.transaction-status--confirmed');
+      const timestampAfterAction = new Date();
+      loadingTimes = timestampAfterAction - timestampBeforeAction;
+    },
+  );
+  return loadingTimes;
+}
+
+async function main() {
+  const { argv } = yargs(hideBin(process.argv)).usage(
+    '$0 [options]',
+    'Run a page load benchmark',
+    (_yargs) =>
+      _yargs.option('out', {
+        description:
+          'Output filename. Output printed to STDOUT of this is omitted.',
+        type: 'string',
+        normalize: true,
+      }),
+  );
+
+  const results = {};
+  results.loadAccount = await loadAccount();
```

Comment:
would it be helpful if we renamed this prop and the function name to loadNewAccount?

Reply:
would it be helpful if we renamed this prop and the function name to loadNewAccount or even createAndLoadNewAccount?